tests/reftests/autopin.test

```diff
@@ -247,14 +245,10 @@ The following system packages will first need to be installed:
 [WARNING] Opam package foo-format.2.0 depends on the following system package that can no longer be found: some-depext
 The following actions will be performed:
 === remove 1 package
-  - remove    foo-format 2.0 (pinned)
-=== recompile 1 package
-  - recompile foo-core   2.0 (pinned) [uses foo-format]
```

Comment:
note: This recompile is a consequence of the previous syncing, that marked `foo-core` to reinstall.